### PR TITLE
[zh-cn]Sync container-runtimes.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Sync upstream
```yaml
content/zh-cn/docs/setup/production-environment/container-runtimes.md
```
en:
image

### CODE REVIEW

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -515,6 +518,18 @@ For `cri-dockerd`, the CRI socket is `/run/cri-dockerd.sock` by default.
...
+<!--
+#### Overriding the sandbox (pause) image {#override-pause-image-cri-dockerd}
+
+The `cri-dockerd` adapter accepts a command line argument for
+specifying which container image to use as the Pod infrastructure container (“pause image”).
+The command line argument to use is `--pod-infra-container-image`.
+-->
+#### 重载沙箱（pause）镜像   {#override-pause-image-cri-dockerd}
+
+`cri-dockerd` 适配器能够接受指定用作 Pod 的基础容器的容器镜像（“pause 镜像”）作为命令行参数。
```
Comment (Michelle951) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:530]:
```suggestion
`cri-dockerd` 适配器支持通过命令行参数指定将哪个容器镜像用作 Pod 的基础容器（“pause 镜像”）。
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 2]
```diff
@@ -61,9 +61,8 @@ v1.24 之前的 Kubernetes 版本直接集成了 Docker Engine 的一个组件
...
 <!--
 You can read
-[Check whether Dockershim deprecation affects you](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/)
-to understand how this removal might
-affect you. To learn about migrating from using dockershim, see
+[Check whether Dockershim removal affects you](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)
+to understand how this removal might affect you. To learn about migrating from using dockershim, see
 [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/).
 -->
 你可以阅读[检查 Dockershim 移除是否会影响你](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)以了解此删除可能会如何影响你。
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:68]:
```suggestion
你可以阅读[检查 Dockershim 移除是否会影响你](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)以了解此移除可能会如何影响你。
```

**Comment by tengqm** [comment]
The duplicated section should be removed in the English upstream.
/hold

**Pull request closed without merging**